javascripts/content_scripts/post_new_link.js

```diff
@@ -24,9 +31,6 @@ document.addEventListener( "contextmenu", function(evt) {
   }
 });
 
-// Variable used to indicate whether there is a pending pending operation
-var pendingPost = false;
```

Comment:
Yes. It's used in `javascripts/content_scripts/posting_button.js`. And, it's still in the global scope. Check the commit once again. I've just moved the initialization up.